File: C++11/백준/Bronze/18414. X に最も近い値 （The Nearest Value）/X に最も近い値 （The Nearest Value）.cc
#include <stdio.h>

int main(){
    int a,b,c;
    scanf("%d %d %d",&a,&b,&c);
    int min = a<b?a:b;
    min = min<c?min:c;
    int max = a>b?a:b;
    max = max>c?max:c;
    printf("%d",a+b+c-min-max);
}

File: Java/백준/Bronze/18414. X に最も近い値 （The Nearest Value）/X に最も近い値 （The Nearest Value）.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] a = new int[3];
        for (int i = 0; i<3; i++){
            a[i] = in.nextInt();
        }
        int min = Math.min(Math.min(a[0],a[1]),a[2]);
        int max = Math.max(Math.max(a[0],a[1]),a[2]);
        System.out.println(a[0]+a[1]+a[2]-min-max);
    }
}